systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -113,6 +125,40 @@ public void testDeployUndeploy() {
         testDockerImagesForKafkaConnect();
     }
 
+    @Test
+    @JUnitGroup(name = "regression")
+    @ConnectS2ICluster(name = CONNECT_CLUSTER_NAME, connectConfig = CONNECT_CONFIG_CONVERTER_SCHEMAS_DISABLED)
```

Comment:
Is this somehow marked as OpenShft only? I think we should add it to this test as it won't work on Minikube.

Reply:
yes, this definitely makes sense